jobserver/issues.py

```diff
@@ -133,7 +133,7 @@ def create_switch_repo_to_public_request(repo, user, github_api):
     body = f"""
     The [{repo.name}]({repo.url}) repo is ready to be made public.
 
-    This repo has been checked and approved by {user.name}.
+    This repo has been checked and approved by {user.name} - @{user.username}.
```

Comment:
I hesitate to be this pedantic, but do we need name _and_ username? Louis suggested the latter instead of the former in the issue. If you think "Yes" then I accept your answer without further comment 🙂

Reply:
I can think of a few people who have less obvious usernames than you or I do. I figured it'd save someone the effort of "who on earth is madwort or bloodearnest?" etc.

Reply:
Chiming in to say we have usernames outside the tech team that don't relate to full names in any way is the main reason I started on the journey to enforce full names! (eg who on earth is <alphanumeric string>, they're running jobs so _someone_ must know who they are?!)

Reply:
Very reasonable!